packages/datadog-plugin-mysql/src/index.js

```diff
@@ -22,6 +24,12 @@ class MySQLPlugin extends DatabasePlugin {
         'out.port': dbConfig.port
       }
     })
+    if (this.config.dbmPropagationMode !== 'disabled') {
+      if (sql[0].sql !== undefined) {
+        const key = 'sql'
+        sql[0][key] = this.injectDbmQuery(sql[0].sql)
```

Comment:
I have an opened PR, that adds more test to test other plugin related with pg, mysql and mysql2, which need the original not modified sql: https://github.com/DataDog/dd-trace-js/pull/2565

I think that this change will break the test that are pending to merge in the other PR. The solution proposed for `pg` is to send also another property `originalQuery` in the channel, that the plugin is not going to modify, and use it in analyzer: 
https://github.com/DataDog/dd-trace-js/blob/a3c60dff87685b3ad08ea3287253137d565ac382/packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js#L7

Reply:
That wouldn't work if the publisher was the actual library. We should come up with a solution that will continue to work in the future.